test/unit/tools/pep8.rb

```diff
@@ -0,0 +1,22 @@
+module Test
+  module Quality
+    module Tools
+      # Test for the 'bigfiles' tool within the quality gem
+      module Pep8
+        def pep8_expected_args
+          'fake1.py'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.